src/js/components/inspectors/Property.tsx

```diff
@@ -71,7 +73,7 @@ function mapStateToProps(reduxState: State, ownProps: OwnProps): StateProps {
   const field = getIn(state, path + '.field');
   const scaleName = scale && getInVis(reduxState, 'scales.' + scale + '.name');
 
-  const value = getIn(state, path);
+  const value = ownProps.value === undefined ? getIn(state, path) : null;
```

Comment:
if you read this and then read line 79, this change appears to amount to this effect: return ownProps.value if it's not undefined, else get it from the store. if the store value is null, return undefined. If that's what we want, we should try rewriting it as a less complex expression

Reply:
oh yeah i missed that. thnx!